<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>610</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>610</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NIIAS</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="errorLines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignLeading</set>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>521</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Error log:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>371</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSensorData">
     <attribute name="title">
      <string>Sensor data</string>
     </attribute>
     <widget class="QListWidget" name="sensorLines">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>291</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignLeading</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabResult">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Result</string>
     </attribute>
     <widget class="QGroupBox" name="bySensorGroupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>9</y>
        <width>571</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>By sensor</string>
      </property>
      <widget class="QListWidget" name="sensorPickList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>191</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
      </widget>
      <widget class="QLabel" name="sensorTitleLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>29</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No sensor info</string>
       </property>
      </widget>
      <widget class="QLabel" name="sensorChasisFormulaLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>49</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No chasis formula</string>
       </property>
      </widget>
      <widget class="QListWidget" name="sensorInfoLines">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>331</width>
         <height>111</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::IgnoreAction</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignLeading</set>
       </property>
      </widget>
      <widget class="QLabel" name="sensorAverageVelocityLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No average velocity</string>
       </property>
      </widget>
      <widget class="QLabel" name="sensorVelocityDeviationLabel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>461</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No velocity deviation</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="timeGroupBox">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>110</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Time</string>
       </property>
       <widget class="QRadioButton" name="timeRelativeRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>23</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Relative</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="timeAbsoluteRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>53</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="overallGroupBox">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>249</y>
        <width>571</width>
        <height>81</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Overall</string>
      </property>
      <widget class="QLabel" name="overallChasisFormulaLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>25</y>
         <width>551</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No chasis formula</string>
       </property>
      </widget>
      <widget class="QLabel" name="overallLengthFormulaLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>51</y>
         <width>551</width>
         <height>21</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>No length formula</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewGroupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>389</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QGraphicsView" name="graphicsBackView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>61</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
     <property name="interactive">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="transformationAnchor">
      <enum>QGraphicsView::NoAnchor</enum>
     </property>
     <property name="viewportUpdateMode">
      <enum>QGraphicsView::FullViewportUpdate</enum>
     </property>
     <property name="rubberBandSelectionMode">
      <enum>Qt::ContainsItemShape</enum>
     </property>
     <property name="optimizationFlags">
      <set>QGraphicsView::IndirectPainting</set>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>▷</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>□</string>
     </property>
    </widget>
    <widget class="QLabel" name="viewStats">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="counterList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open Sensor data...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
